### PR TITLE
fix(ingestion): fix hive database ingestion recipe

### PR DESCRIPTION
**Pull request by chinmay-bhat**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Review by shirshanka — APPROVED** [review]
LGTM!

File: metadata-ingestion/tests/unit/test_hive_source.py [hunk 1]
```diff
@@ -16,6 +16,8 @@ def test_hive_configuration_get_identifier_with_database():
         "scheme": "hive+https",
     }
     hive_config = HiveConfig.parse_obj(config_dict)
-    expected_output = f"{test_db_name}.{test_schema_name}.{test_table_name}"
-    output = hive_config.get_identifier(schema=test_schema_name, table=test_table_name)
+    expected_output = f"{test_db_name}.{test_table_name}"
+    ctx = PipelineContext(run_id="test")
+    hive_source = HiveSource(config_dict, ctx)
```
Comment (hsheth2) [inline metadata-ingestion/tests/unit/test_hive_source.py:21]:
```suggestion
    hive_source = HiveSource(hive_config, ctx)
```

Comment (chinmay-bhat) [reply metadata-ingestion/tests/unit/test_hive_source.py]: updated

**Review by shirshanka — APPROVED** [review]
LGTM!

**Pull request merged**